### DISCUSSION TITLE
Cant install

### DISCUSSION THREAD

**Discussion by Abid-29** [TheWidlarzGroup/react-native-video#4152] [Q&A] [answered]
### What are you trying to do?

I am unable to install the latest version. I first used 5.2.1 but in it there is no button for fullscreen in android. Now when I update to 6.5.0 and follow the steps on documentation, it still does not complete the installation and instead keep on installing the app for ages, please help

### What have you tried?

Upgraded from 5.2.1 to 6.5.0 and followed the steps on the documentation. The installation does not complete and instead keeps on installing the app for ages.

### Version

v6

### Platforms

Android

**Comment by Abid-29** [comment]
It works perfectly fine on 5.2.1, the only reason I am updating it to a higher version is so that I can have a fullscreen option in android, if there is another way, please let me know

**Comment by freeboub** [comment, marked as answer]
Looks like an issue on app side, but without more information, we cannot help
